recipes/picohttpparser/all/patches/cci.20230804-0001-add-cmake.patch

```diff
@@ -0,0 +1,25 @@
+diff --git a/CMakeLists.txt b/CMakeLists.txt
```

Comment:
It seems like the project is abandoned, but still I'll ask you to send this CMake support the upstream first.

To use here, please, do not use as a patch, but use export_sources instead. Take a look in `tgc` recipe.